src/AbstractFixer.php

```diff
@@ -45,6 +46,13 @@ public function __construct()
         }
     }
 
+    public function fix(\SplFileInfo $file, Tokens $tokens)
```

Comment:
Correct, the new abstract method is already a BC break. I think it should not be abstract and have an empty body instead. Side-projects that extend this class will still override `fix` so it should not break.